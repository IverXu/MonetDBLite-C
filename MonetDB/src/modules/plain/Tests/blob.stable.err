stderr of test 'blob` in directory 'modules/plain` itself:

#> Mtest:	other blob
#> Mtest:	./blob blob
#> Mlog:	Mserver -yaccsilent -d8 -home /ufs/manegold/Current/monet/IRIX6.3 -db modules_plain -single /ufs/manegold/Current/monet/IRIX6.3/modules/plain/blob.mil
