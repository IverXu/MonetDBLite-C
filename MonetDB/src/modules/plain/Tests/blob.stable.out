stdout of test 'blob` in directory 'modules/plain` itself:

#> Mtest:	other blob
#> Mtest:	Mtimeout -timeout 420 ./blob blob
#> Mlog:	Mtimeout -timeout 210 Mserver -modulesilent -yaccsilent -d8 -db modules_plain blob.mil
# Monet Database Server V4.1
# Copyright (c) 1993-1999, CWI & SION. All rights reserved.
# compiled for IRIX646.5/32bit on Fri Jan 14 01:04:33 MET 2000.
#script:blob.mil

loaded;
#---------------------------------#
# module	| usage_count	  #
#---------------------------------#
[ "algebra",	  1		  ]
[ "arith",	  1		  ]
[ "bat",	  1		  ]
[ "kernel",	  1		  ]
[ "str",	  1		  ]
[ "sys",	  1		  ]
[ "trans",	  1		  ]

Try: modules;                - for other modules.
     ls;                     - for persistent bat listing.
     sigs("<module>")        - for available commands per module.
     help("<command>")       - for help one-liner for a command.

#You are system administrator (adm)!

#line 46 "blob.mx"
setoid(oid(20000000));
module(blob);
bl := blob("(5:11 FC E2 A0 0A)");
bl.print;
[ (5:11 FC E2 A0 0A) ]
bl2 := toblob("Hello World");
bl2.print;
[ (12:48 65 6C 6C 6F 20 57 6F 72 6C 64 00) ]
tostr(bl2).print;
[ "Hello World" ]

quit;
